extensions/sonarqube/overview.md

```diff
@@ -1,50 +1,59 @@
 
-### Branch and Pull Request analysis
-Whatever type of source repository you are analysing, when a build is run on a branch of your project, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube:
+### Automatically Analyze Branches and Decorate Pull Requests (SonarQube Commercial Editions)
+This extension allows automatic analysis of all branches and pull-requests which enables early discovery of bugs and security vulnerabilities prior to a merge. Branches and PRs get their own Quality Gate status and results are pushed right to your project space. 
 
 ![Branches](img/branches.png)
 
-If you configure your build definition as a build validation for pull requests of that project (this can be done on "Branch policies"), SonarQube will also analyze the code changes and decorate the pull request with comments and overall status so that you can merge with confidence:
+It's easy to add non-disruptive code quality checks right into your Azure DevOps workflow. Simply add SonarQube to your build pipeline definition and you're on your way to only promoting quality code. SonarQube analyzes the code changes and decorates pull requests with comments and overall status -> _merge with confidence!_
+
+![Build Config](img/Build-pipeline.png)
+
+When a build is run on a branch, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube.
 
 ![PR-Decoration](img/pull-request-decoration.png)
 
 **Important note**: to activate pull request decoration, you must specify a user token in the "General Settings > Pull Requests" administration page of your project in SonarQube.
 
-### Quality Gate Status
-The **Publish Quality Gate Result** task waits for the analysis report to be consumed by the SonarQube in order to flag the build job with the Quality Gate status. The Quality Gate is a major, out-of-the-box, feature of SonarQube. It provides the ability to know at each analysis whether an application passes or fails the release criteria. In other words it tells you at every analysis whether an application is ready for production "quality-wise".
+## Additional Highlights
+### Seamless integration with .NET solutions
+C# and VB.NET analysis is simple and straightforward and only requires adding the **Prepare Analysis Configuration** and **Run Code Analysis** tasks to your build definition.
 
-Example of a passing Quality Gate:
-![Passed Qualiy Gate](img/sq-analysis-report-passed.png)
+### Easy setup for Maven and Gradle projects
+For Java, analyzing your source code is also very easy. It only requires adding the **Prepare Analysis Configuration** task and checking the **Run SonarQube Analysis** option in the 'Code Analysis' panel in your Maven or Gradle task.
 
-Example of a failing Quality Gate:
-![Failed Qualiy Gate](img/sq-analysis-report-failed.png)
+## SonarQube Azure DevOps Marketplace Extension Details
+This extension provides the following features:
```

Comment:
Talking about "features" for the endpoint and the tasks looks weird to me.
Also, I'm not sure we should repeat Branch and Pull Request analysis among those bullet points.
Or maybe:

> This extension provides Branch and **Pull Request** analysis with:...

?